lib/models/BucketInfo.ts

```diff
@@ -13,7 +13,7 @@ import { areTagsValid, BucketTag } from '../s3middleware/tagging';
 // WHEN UPDATING THIS NUMBER, UPDATE BucketInfoModelVersion.md CHANGELOG
 // BucketInfoModelVersion.md can be found in documentation/ at the root
 // of this repository
-const modelVersion = 14;
+const modelVersion = 16;
```

Comment:
do we need to skip one version?

Reply:
15 seems already defined

Reply:
yes, the bump in the ts file was missed, I wanted to be consistent with what is documented.